### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by HengyueLi** [confunguido/pandoc_tex_nos#1]
Firstly, thanks for your effort! I found this from your [blog](https://confunguido.github.io/blog/20190614_pandoc_filters.html).

Recently, I use `pandoc` with `pandoc-crossf`, everything looks almost perfect except the equation numbering. Can you give me an example of how to use this project?

**Comment by ericrpatton** [comment]
Yes I agree, would it be possible to include an example on how to use this program?
